src/lifecycle.js

```diff
@@ -111,7 +111,7 @@ function addAttributeToPropertyLinks (target, component) {
   }
 
   for (var attribute in componentAttributes) {
-    if (hasOwn(componentAttributes, attribute) && !hasOwn(target, attribute)) {
+    if (hasOwn(componentAttributes, attribute) && !hasOwn(target, camelCase(attribute))) {
```

Comment:
It'd be an unnecessary micro-optimisation at this point probably, but my only concern here is that we're doubling up on the `camelCase()` calls. Thoughts? I'm happy to just let it go through; I doubt there'll be any issues with it. Easy fix if there is.